packages/doc/content/components/components/card/plancard-native.mdx

```diff
@@ -11,7 +11,7 @@
   >
     <PlanCard.Subtitle>Get access to</PlanCard.Subtitle>
     <PlanCard.List>
-      <PlanCard.ListItem icon={MapPin} text="2.900 gyms and studios" />
+      <PlanCard.ListItem color="#6B6B78" icon={MapPin} text="2.900 gyms and studios" />
```

Comment:
It's better to follow the preference order below, from left to right:

| theme color | token | hex |
| ------------ | ------ | --- |
| `text.secondary` | `deep` | `#6B6B78 ` |